File: daemons/tests/get_mcid.py
#!/usr/bin/env python

import SOAPpy

URL = 'http://mcid.internal:1080/mcid'
NS  = 'http://www.medcommons.net/mcid'

mcid_generator = SOAPpy.SOAPProxy(URL, namespace=NS)

if __name__ == '__main__':
    print mcid_generator.next_mcid()

File: java/identity/mcid_server/get_mcid.php
#!/usr/bin/env php
<?php

    $URL = 'http://mcid.internal:1080/mcid';
    $NS = 'http://www.medcommons.net/mcid';

    $client = new SoapClient(null, array('location' => $URL, 'uri' => $NS));

    $mcid = $client->next_mcid();

    echo $mcid;

?>

